### DISCUSSION TITLE
Request more than 5 var ptms

### DISCUSSION THREAD

**Discussion by BenoitBe** [vdemichev/DiaNN#1215] [Q&A]
Hello Vadim,

I have again a new question :)

Is it possible to request more than 5ptms?

I tried and I have this message: "WARNING: only the first 5 variable modifications specified will be searched for : 1"

In addition, because I am focusing on methylation (epigenetic project), I have wrote this:
"
--var-mod UniMod:1000,28.0313,K
--var-mod UniMod:1001,28.0313,R
"

Is it possible to write: "--var-mod UniMod:1000,28.0313,KR"? Meaning the same modif for R and/or K?

Thanks a lot in advance,

Benoit

**Comment by vdemichev** [comment]
Hi Benoit,

Thanks for pointing out, I will take a look if this limitation can be removed.

`Is it possible to write: "--var-mod UniMod:1000,28.0313,KR"?`
Yes, absolutely, can also name the modification in whatever way you like, does not need to reference UniMod.

Best,
Vadim
